src/opencl_common.c

```diff
@@ -291,8 +291,9 @@ static char *remove_spaces(char *str) {
 
 static char *opencl_driver_info(int sequential_id)
 {
-	static char ret[64];
-	char dname[MAX_OCLINFO_STRING_LEN], tmp[64], set[64];
+	static char ret[64 + MAX_OCLINFO_STRING_LEN];
+	static char output[64 + MAX_OCLINFO_STRING_LEN * 2];
```

Comment:
@magnumripper Does this look OK to you? I had to introduce a new variable to eliminate "circular dependencies` in the buffer sizes of involved variables.

Reply:
I guess to be perfect it should be `output[sizeof(tmp) + sizeof(dname)]`. Also I think the name `ret` is because we are (or was) returning that one so these changes make that name a bit misleading.

Reply:
Ack. Will fix soon.